tests/backends/test_ddp.py

```diff
@@ -68,3 +72,36 @@ def test_multi_gpu_model_ddp_fit_test(tmpdir, cli_args):
     model_outs = result['result']
     for out in model_outs:
         assert out['test_acc'] > 0.90
+
+
+# START: test_cli ddp test
+@pytest.mark.skipif(os.getenv("PL_IN_LAUNCHER", '0') == '1', reason="test runs only in DDPLauncher")
+def internal_test_cli(tmpdir, args=None):
+    """
+    This test verify we can call function using test_cli name
+    """
+
+    return 1
+
+
+@pytest.mark.skipif(torch.cuda.device_count() < 2, reason="test requires multi-GPU machine")
+def test_cli(tmpdir):
+    DDPLauncher.run_from_cmd_line("--max_epochs 1 --gpus 2 --accelerator ddp", internal_test_cli, tmpdir)
+    # load the results of the script
+    result_path = os.path.join(tmpdir, 'ddp.result')
+    result = torch.load(result_path)
+    # verify the file wrote the expected outputs
+    assert result['status'] == 'complete'
+    assert str(result['result']) == '1'
+# END: test_cli ddp test
+
+
+@pytest.mark.skipif(torch.cuda.device_count() < 2, reason="test requires multi-GPU machine")
+@DDPLauncher.run("--max_epochs [max_epochs] --gpus 2 --accelerator [accelerator]",
+                 max_epochs=["1"],
+                 accelerator=["ddp", "ddp_spawn"])
```

Comment:
```suggestion
@DDPLauncher.run(gpus=[2],
                 max_epochs=[1],
                 accelerator=["ddp", "ddp_spawn"])
```
feel like this would be easier to read assuming that the kwargs are the same as CLI keywords
seeing some CMMD makes me think what a hell the case here, which command shall I use...